doc/src/manual/noteworthy-differences.md

```diff
@@ -107,6 +107,10 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Like many languages, Julia does not always allow operations on vectors of different lengths, unlike
     R where the vectors only need to share a common index range.  For example, `c(1, 2, 3, 4) + c(1, 2)`
     is valid R but the equivalent `[1, 2, 3, 4] + [1, 2]` will throw an error in Julia.
+  * Julia allows an optional trailing comma when that comma does not change the meaning of code.
+   This can cause confusion among R users when indexing into arrays. For example, `x[1,]` in R
```

Comment:
Not sure if this affects rendering, but this is indented one space less than one would expect.

Reply:
I fixed it directly.